spark/spark-spa/src/main/java/com/thoughtworks/go/spark/spa/spring/VelocityTemplateEngine.java

```diff
@@ -51,6 +52,8 @@ public String render(ModelAndView modelAndView) {
             VelocityContext context = initialContextProvider.getVelocityContext((Map) model, controller, modelAndView.getViewName());
             StringWriter writer = new StringWriter();
             template.merge(context, writer);
+            Object meta = context.get("meta");
+            context.put("meta", new Gson().toJson(meta));
```

Comment:
`Gson` instances are slightly heavy object to construct. Consider putting it in a static variable.